docs/self-managed/setup/deploy/amazon/amazon-eks/eksctl.md

```diff
@@ -1,13 +1,15 @@
 ---
 id: eks-eksctl
-title: "Deploy an EKS cluster with eksctl"
+title: "Deploy an EKS cluster with eksctl (fast-try)"
```

Comment:
not sure about the `(fast-try)` in the title.

Maybe we could utilize tags - https://docusaurus.io/docs/2.x/create-doc#doc-tags
and then maybe something like `quick guide`.

Reply:
maybe just `quickstart`.

Like with manuals, you usually have the complete manual and the quickstart guide